packages/dx-react-grid-bootstrap4/src/templates/table-header-cell/content.jsx

```diff
@@ -3,14 +3,18 @@ import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 export const Content = ({
-  column, children, align, className, ...restProps
+  column, children, align, className, style, ...restProps
 }) => (
   <div
     className={classNames({
       'w-100 d-flex flex-row align-items-end': true,
       'justify-content-center': align === 'center',
       'justify-content-end': align === 'right',
     }, className)}
+    style={{
+      minWidth: 0,
+      ...style,
+    }}
```

Comment:
We should avoid inline styles in bs4 / mui packages. We can create custom classes with these styles in the `dx-styles` package and apply class instead of styles.